addons/delivery/models/product_template.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    @api.multi
+    def write(self, vals):
+        #TODO FixMe, onchange of template list_price change the fixed price of the delivery method and update price rules based on fixed price.
```

Comment:
Hello @dmo-odoo ,
Actually the problem is inverse function does not call the write method of carrier. and if we can use directly write in inverse function it will give me trace-back. so that we have called method over here. Thanks